fdbserver/tester.actor.cpp

```diff
@@ -253,6 +253,23 @@ std::vector<std::string> getOption(VectorRef<KeyValueRef> options, Key key, std:
 	return defaultValue;
 }
 
+std::vector<int> getOption(VectorRef<KeyValueRef> options, Key key, std::vector<int> defaultValue) {
```

Comment:
Can you add a comment detailing what this function does (and maybe an example)? It's a bit hard to decipher from reading the code

Reply:
It is a helper method to parse comma-separated integer list in test specs. The comment is in the function declaration in workloads.action.h above.